.gitignore

```diff
@@ -47,6 +47,9 @@ npm-debug.log
 yarn-debug.log
 yarn-error.log
 
+# Ignore files generated by npm pack
+/*.tgz
+
```

Comment:
Not totally related to this PR, but I use `npm pack` a lot to test this package in other repos and I don't want to accidentally commit the .tgz.